src/odemis/acq/stream/test/stream_sparc1_test.py

```
@@ -0,0 +1,93 @@
+
+logging.basicConfig(format="%(asctime)s  %(levelname)-7s %(module)s:%(lineno)d %(message)s")
+logging.getLogger().setLevel(logging.DEBUG)
+
+CONFIG_PATH = os.path.dirname(odemis.__file__) + "/../../install/linux/usr/share/odemis/"
+SPARC_CONFIG = CONFIG_PATH + "sim/sparc-pmts-sim.odm.yaml"
+
+
+class SPARCTestCase(BaseSPARCTestCase):
+    """SPARC v1 tests"""
+    simulator_config = SPARC_CONFIG
+    capabilities = {"ar", "spec", "monochromator"}
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)